src/components/serviceNavigator/ServiceNavigator.js

```diff
@@ -3,7 +3,7 @@ import ServiceCategoryList from './ServiceCategoryList'
 import ServiceCategorySectionList from './ServiceCategorySectionList'
 import ServiceCategoryService from '../../services/ServiceCategoryService'
 import serviceCategories from '../../mock/data/service_categories.mock.json'
-
+import serviceCards from './ServiceCards'
```

Comment:
A maybe unnecessary import? Haven't tested and not familiar with js but could be

Reply:
This is definitely not necessary